Articles of Association.md

```diff
@@ -0,0 +1,66 @@
+1. Name
+    1. The name of the association is "Feather Association"
+2. Purpose
+    1. The purpose of the association is
+        1. The promotion of feather
```

Comment:
Might wanna further specify what we mean by "feather"

Reply:
We'd want to include Feather as well as all its subprojects in this definition—i.e, the organization as a whole.

Reply:
👍